docs/Aggregations.md

```diff
@@ -400,11 +400,14 @@ Note that these operators apply only to numeric values and numeric sub expressio
 | Function | Description                                                  | Example            |
 | -------- | ------------------------------------------------------------ | ------------------ |
 | geodistance(field,field)        | Return distance in meters.    | `geodistance(@field1,@field2)`       |
-| geodistance(field,lon,lat)      | Return distance in meters.    | `geodistance(@field,1.2,-3.4)`       |
 | geodistance(field,"lon,lat")    | Return distance in meters.    | `geodistance(@field,"1.2,-3.4")`     |
-| geodistance(lon,lat,field)      | Return distance in meters.    | `geodistance(1.2,-3.4,@field)`       |
+| geodistance(field,lon,lat)      | Return distance in meters.    | `geodistance(@field,1.2,-3.4)`       |
 | geodistance("lon,lat",field)    | Return distance in meters.    | `geodistance("1.2,-3.4",@field)`     |
 | geodistance("lon,lat","lon,lat")| Return distance in meters.    | `geodistance("1.2,-3.4","5.6,-7.8")` |
+| geodistance("lon,lat",lon,lat)  | Return distance in meters.    | `geodistance("1.2,-3.4",5.6,-7.8)`   |
+| geodistance(lon,lat,field)      | Return distance in meters.    | `geodistance(1.2,-3.4,@field)`       |
+| geodistance(lon,lat,"lon,lat")  | Return distance in meters.    | `geodistance(1.2,-3.4,"5.6,-7.8")`   |
+| geodistance(lon,lat,"lon,lat")  | Return distance in meters.    | `geodistance(1.2,-3.4,5.6,-7.8)`     |
```

Comment:
I think that it would be better to just say geodistance("lon,lat", "lon,lat"), geodistance("lon,lat", lon, lat), geodistance(lon, lat, "lon,lat"), geodistance(lon, lat, lon, lat) and the fact that you can use fields in this format is derived. Also, because fields can be string or number, writing 'field' is not 100% correct. @K-Jo what do you think?

Reply:
@MeirShpilraien I do believe that this table exploded and we should simplify.
geoDistance(string coordinate1, string coordinate2)
geoDistance(string coordinate1, double longitude, double latitude)
geoDistance(double longitude, double latitude, String) (although not sure why we created this)
geoDistance(double longitude1, double latitude1, double longitude2, double latitude2)
Where string should represent a coordinate separating longitude and latitude by a comma e.g. "5.6,-7.8".  This is also the way the field within the hash should be modelled to represent a coordinate.  Hence you can also pass the field for `@field_coordinate` to the coordinate string values.

Also I would choose the example coordinates to be a brewery.